library/data/bitvec.lean

```diff
@@ -136,20 +136,49 @@ end comparison
 section conversion
   variable {α : Type}
 
+  @[reducible]
+  protected def list_of_nat : ℕ → ℕ → list bool
+  | 0        x := list.nil
+  | (succ n) x := list_of_nat n (x / 2) ++ [to_bool (x % 2 = 1)]
+
   protected def of_nat : Π (n : ℕ), nat → bitvec n
```

Comment:
You're right, list_of_nat doesn't actually need to be reducible and we don't need to duplicate the definition of list_of_nat